crud/delete.lua

```diff
@@ -51,10 +54,58 @@ local function call_delete_on_router(space_name, key, opts)
         return nil, DeleteError:new("Space %q doesn't exist", space_name), true
     end
 
+    local primary_index = space.index[0]
+    local space_format = space:format()
```

Comment:
You are not using space_format variable below. Maybe pass it to the get_keys_fieldno_map function directly, without creating a variable?

Reply:
> You are not using space_format variable below. 

It's true.
But I think it is better to declare most used variables (like space format, index etc) on top of function's body for future use.